UTAT_Compression_Algorithm/C/main.c

```diff
@@ -2,15 +2,14 @@
 
 #include "compression.h"
 
+int main()
+{
 
-
-int main(){
-
-    //Unit test for Prediction Calculation
+    // Unit test for Prediction Calculation
     printf("Creating a local difference vector of size 5 with all values equal to 5\n");
     printf("Creating a weights vector of size 5 with all values equal to 3\n ");
-    gsl_vector* local = gsl_vector_calloc(5);
```

Comment:
do you prefer doing `int *arr` instead of `int* arr`? I like `int* arr`, because array is of type `int*`, so I like to keep the int and * together.

Reply:
I've always written it as `int *arr` because that's how I've been taught